Week-4/Ex1.js

```diff
@@ -0,0 +1,58 @@
+function getNumber() {
+    return Math.floor(Math.random() * 100);
+}
+
+function getPromise(executionFunction) {
+    'use strict';
```

Comment:
As discussed use ES6 classes way as well to get deeper understanding

Reply:
Sure. I will give it one more attempt.